### PR TITLE
Add Player:Ping

### PR DESCRIPTION
**Pull request by green1052**
add get player ping func

### CODE REVIEW

**Review by joker-119 — APPROVED** [review]
How was this not already a thing?

**Review by joker-119 — CHANGES_REQUESTED** [review]
Fix the build errors.

**Comment by BuildBoy12** [comment]
This should also be a property instead of a method.

**Comment by green1052** [comment]
like this?

```
public int Ping 
{

    get 
    {
        return LiteNetLib4MirrorServer.GetPing(Connection.connectionId);
    }

}

**Comment by BuildBoy12** [comment]
`public int Ping => LiteNetLib4MirrorServer.GetPing(Connection.connectionId)`; is identical to what you posted.

**Comment by green1052** [comment]
https://github.com/Exiled-Team/EXILED/pull/501/commits/136b395a705112cac352cf300d7855cf56f04d50 thx

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1353,6 +1354,12 @@ public void ShowHint(string message, float duration = 3f)
 
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
+        
+        /// <summary>
+        /// Get player ping.
```
Comment (iopietro) [inline Exiled.API/Features/Player.cs:1359]:
```suggestion
        /// Gets player's ping.
```

**Review by iopietro — APPROVED** [review]

**Comment by iopietro** [comment]
Like Joker said, fix build errors.

**Review by BuildBoy12 — APPROVED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**